package.json

```diff
@@ -200,12 +225,31 @@
                 "category": "Azure Resource Manager Tools",
                 "title": "Open Parameter File",
                 "command": "azurerm-vscode-tools.openParameterFile",
-                "enablement": "editorLangId==arm-template"
+                "enablement": "azurerm-vscode-tools-hasParamFile",
+                "$enablement.comment": "Shows up when it's a template file, but only enabled if there is an associated param file"
             },
             {
+                "$comment": "============= Parameter file commands =============",
                 "category": "Azure Resource Manager Tools",
-                "title": "Reset Global State",
-                "command": "azurerm-vscode-tools.resetGlobalState"
+                "title": "Open Template File",
+                "command": "azurerm-vscode-tools.openTemplateFile",
+                "enablement": "azurerm-vscode-tools-hasTemplateFile",
+                "$enablement.comment": "Shows up when it's a param file, but only enabled if there is an associated template file"
+            },
+            {
+                "$comment": "============= Parameter file commands =============",
+                "category": "Azure Resource Manager Tools",
+                "title": "Add all missing parameters",
```

Comment:
>a [](start = 30, length = 1)

Nit: Should this be title cased? Same with the command title below?

Reply:
fixed thx